grid_map_demos/src/opencv_demo_node.cpp

```diff
@@ -19,25 +16,26 @@ int main(int argc, char** argv)
   GridMap map({"original", "elevation"});
   map.setFrameId("map");
   map.setGeometry(Length(1.2, 2.0), 0.01);
-  ROS_INFO("Created map with size %f x %f m (%i x %i cells).",
+  ROS_INFO(
```

Comment:
ROS?

Reply:
This file is not ported into this PR, it is just linted.
This will be addressed in an upcoming PR.